Package.swift

```diff
@@ -27,6 +27,10 @@ let package = Package(
             targets: ["StreamChatTestMockServer"]
         ),
     ],
+    dependencies: [
```

Comment:
Why not embed these in the test tools / mock server?

Reply:
Great idea, let's try that. This will increase the PR grammatically, but that will be just embedded libs.